padinfo/core/transforminfo.py

```diff
@@ -0,0 +1,23 @@
+from padinfo.core.find_monster import findMonsterCustom2
+from padinfo.core.id import get_monster_misc_info
+
+
+async def perform_transforminfo_query(dgcog, raw_query, beta_id3):
+    db_context = dgcog.database
+    mgraph = dgcog.database.graph
+    m, err, debug_info = await findMonsterCustom2(dgcog, beta_id3, raw_query)
+    altversions = mgraph.process_alt_versions(m.monster_id)
+
+    for mid in sorted(altversions):
+        if mgraph.monster_is_transform_base_by_id(mid):
+            tfm = mgraph.get_monster(mgraph.get_next_transform_id_by_monster_id(mid))
```

Comment:
can you use transformed_mon, base_mon, etc as variable names, I know we have super abbreviated variable names all over the place in our code but we're trying to improve that moving forward

Reply:
yes can do. as you've guessed I just kind of tried to copy the style of what was already in the codebase

Reply:
lol yeah I figured